src/main/webapp/index.css

```diff
@@ -1,3 +1,158 @@
+html, body {
+    width: 100%;
+    height: 100%;
+    background-image: url("https://www.ebsco.com/sites/g/files/nabnos191/files/acquiadam-assets/librarian-pet-ownership-results-blog-image-780.jpg");
+    background-color: gray;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+#logout {
```

Comment:
This is a pr for adding location validation, but it contains all the authentication logic, you need to keep these two changes in separate PRs, please request review once PRs are divided

Reply:
hi Estefany! I've been looking into separating the PRs, but couldn't quite successfully do that—and I'm on a bit of a time crunch today not only with this project but a few other things as well. I know it isn't a good enough solution, but I went back to the files and got rid of all the auth parts (as they exist in the authentications branch anyways), and recommitted, with the group project id.